app/assets/stylesheets/moderators.scss

```diff
@@ -155,14 +155,47 @@
   padding: 30px 10px;
 }
 
+// these styles are to correct height and margin issues in the iframe
+.moderations-actions_panel {
+  min-height: initial;
+  height: 100%;
+  #page-content-inner {
+    margin-bottom: 0;
+    height: 100%;
+  }
+}
```

Comment:
This CSS is because the iframe's HTML contents were technically a page, and I had to overwrite some of the base styling that happens. I also overwrite it via JS in `actions_panel.html.erb`. IMO this is all kind of a temporary hack that probably will be less of a problem if we move to Preact components in the future.